invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides

```diff
@@ -64,3 +64,16 @@
     }
   }
 }
+
+.item{
+  .meta{
+    .mobile.only{
+      .ui.mini.button{
+        width: 100%;
+        margin: 0 0 0.5rem 0;
+        display: block; 
+      }
+    }
+  }
+}
+
```

Comment:
For reviewers: The mobile view uses by default display: inline-block. For it to look good is better to use display: block and, once we had to create the selector, using it for defining a fluid button with margin to separate it from the adjacent ones is cleaner IMO.